app/client/src/features/events/EventDetailsCard/EventDetailsCard.tsx

```diff
@@ -31,16 +34,19 @@ interface Props {
 }
 
 export function EventDetailsCard({ fragmentRef }: Props) {
-    const { title, description } = useFragment(EVENT_DETAILS_CARD_FRAGMENT, fragmentRef);
+    const { title, description, startDateTime, endDateTime } = useFragment(EVENT_DETAILS_CARD_FRAGMENT, fragmentRef);
     const classes = useStyles();
 
+    const startTime = formatDate(startDateTime ? new Date(startDateTime) : new Date(), 'h:mmaa');
+    const endTime = formatDate(endDateTime ? new Date(endDateTime) : new Date(), 'h:mmaa');
```

Comment:
My only nitpick here is that we'd _probably_ want to stick both of these in a React.useMemo since they will be recalcualted on each rerender otherwise, although, they aren't partiuclarly expensive.